SPECS/redis/redis.spec

```diff
@@ -88,6 +86,8 @@ exit 0
 %config(noreplace) %attr(0640, %{name}, %{name}) %{_sysconfdir}/redis.conf
 
 %changelog
+* Mon Oct 18 2021 Neha Agarwal <nehaagarwal@microsoft.com> 5.0.14-1
```

Comment:
While we are here, can we please cleanup the changelog entries so that we are  consistent throughout.
Something like [this](https://github.com/microsoft/CBL-Mariner/blob/1.0/SPECS/kernel/kernel.spec#L561): A blank line separating entries :)